sources/api/ecs-settings-applier/src/main.rs

```diff
@@ -0,0 +1,158 @@
+#[macro_use]
```

Comment:
A simple README here would be nice for `ecs-settings-applier`

Reply:
Done